src/main/webapp/gameBoard.html

```diff
@@ -40,8 +40,10 @@
             <h5 id="headings"> Game Name </h5>
             </br>
  
-            <div id="leaderboard-container">
-                LeaderBoard Elements will be populated Here
+            <div id="leaderboard-container" onload = "loadUsers()">
+                <body onload="loadUsers()">
+                    <ol id="leader-board"></ol>
+                </body>
```

Comment:
Do you need this body tag? This file already has one so maybe you could just put the loadUsers() onload function there

Reply:
I removed it